packages/@aws-cdk-testing/framework-integ/test/aws-lambda/test/integ.permissions.js.snapshot/lambda-permissions.assets.json

```diff
@@ -1,15 +1,15 @@
 {
   "version": "34.0.0",
```

Comment:
Hey, I see the version here was not updated. Just want to confirm if these tests were dry-run? If so, please let me know and I can run these for this PR

Reply:
Hmm, I didn't dry-run..weird. I saw it deploy into my account.

Reply:
That is fine. This happens in some instances. Thank you for confirming that you have deployed and tested these :)